src/components/home/BlurbHome.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+
+const BlurbHome = () => {
+  return (
+    <div className="flex w-[30vw] flex-col items-center">
+      <div>
+        <Image
+          src="/home/blurbforhome.jpeg"
```

Comment:
```suggestion
          src="/home/blurbforhome.jpeg"
```
Import this image, ex: 
Import { Cupcakes } from "@/public/home/cupcakes.jpeg" or something like that and then the source of your image would be src="Cupcakes"

Reply:
I did not know this would work!